content/post/online_hackathon_an_experiment.md

```diff
@@ -0,0 +1,45 @@
+
+The data was arranged accordingly to the instructions mentioned in the problem statement, it took me quite long to design the logic because at first, I thought of it as a Machine Learning problem where I've to detect what to do when a user answers wrong or vice versa but being a web developer I had no clue how ML works. I keep the game running with my logic where I set three categories of questions and allotted an id(-1,0,1) with their marks respectively. Now on every fail or successful attempt, all I have to do is increment/decrement counter and pick my next question accordingly. Maybe it was not the perfect solution for future enhancements but it was complete enough to solve the problem. 
+
+The clock kept ticking, soon it was 9:00 PM when I took a short break for dinner and was back as soon as possible, now the part that was left is the last part, the one in which I have to plot a graph based on user's result. Well, here's one of the reasons justifying my choice of language, I know it could have been possible with any other too but it would have taken longer instead in python I imported matplotlib which is a popular library for plotting graphs using which I plotted a bar graph depicting correct and incorrect answers.
+
+Now I was almost done, all I had to do is to make an executable for my code and share it with the organizer's email. It was 12:30 AM the next day when I reached this scenario and faced some difficulties as being a Linux user I somehow made the executable but was not capable of running it on my machine to check if it was working. Thank you to the team of [Bridgingo](https://bridgingo.com/) who helped me along with my submissions and confirming the working of the same. At last, it was time and I made it through my first hackathon. To be honest I was least interested in the fact that I'll win or lose instead the satisfaction of completing a project over a day and learning a lot of things at once was all I craved for and going through the same was more than enough for me to take a sound sleep. 
+
+I narrated the whole experience to my classmates and friends interested in coding and everyone was all praise for my efforts and the opportunity Bridgingo provided me. It would not have been possible without their motivation and the competitive nature created to keep up with the good work followed by support for submission and understanding the problem. Soon after a few days I received a mail from them, the subject read **Congratulations** which already made me excited and so was the body. The mail was about me being scored in the **Top 50 winners** of the contest named Game Of Codes and along with it was attached my certificate you can see it in the image below. Not only this, they even provided me with 200 \$ credits of Digital Ocean which came as a big surprise to me as being a web developer I am always looking for some professional hosting spaces for running tests. 
```

Comment:
```suggestion
I narrated the whole experience to my classmates and friends interested in coding and everyone was all praise for my efforts and the opportunity Bridgingo provided me. It would not have been possible without their motivation and the competitive nature created to keep up with the good work followed by support for submission and understanding the problem. Soon after a few days I received a mail from them, the subject read **Congratulations** which already made me excited and so was the body. The mail was about me being scored in the **Top 50 winners** of the contest named Game Of Codes and along with it was attached my certificate you can see it in the image below. Not only this, they even provided me with 200 \$ credits of Digital Ocean which came as a big surprise to me as being a web developer I am always looking for some professional hosting spaces for running tests. 
```